force-app/main/default/lwc/courseRegistrationInformation/courseRegistrationInformation.html

```diff
@@ -0,0 +1,29 @@
+<template>
+    <lightning-record-view-form record-id={courseId} object-api-name="Course__c">
+
+        <div class="slds-box slds-theme_shade">
+
+            <div class="slds-text-heading_medium">Informasjon</div>
+
+            <p if:false={courseId}> Vennligst gå tilbake til kursoversikten og velg riktig kurs:
+                https://arbeidsgiver.nav.no/kursoversikt/ </p>
+
+            <div class="slds-text-title_bold">Sted</div>
+            <lightning-output-field variant="label-hidden" field-name="RegistrationPlaceName__c">
+            </lightning-output-field>
+            <p></p>
+
+            <div class="slds-text-title_bold">Tid</div>
+            <lightning-output-field variant="label-hidden" field-name="RegistrationFromDateTime__c">
+            </lightning-output-field>
+            <h1>-</h1>
+            <lightning-output-field variant="label-hidden" field-name="RegistrationToDateTime__c">
+            </lightning-output-field>
```

Comment:
```suggestion
            <div class="slds-text-title_bold slds-p-top_medium">Varighet</div>
            <lightning-output-field variant="label-hidden" field-name="TotalDays__c">
            </lightning-output-field>
```

Lag et formelfelt som heter noe ala "TotalDays__c", som er `RegistrationToDateTime__c  - RegistrationFromDateTime__c` (sluttdato minus startdato). Da får du antall dager kurset går. Vis heller dette istedenfor sluttdato (som kan være tom). Hvis sluttdato er tom, så skal det stå 1 dag i formelfeltet.